src/Controller/Speaker/Create.php

```diff
@@ -48,7 +50,10 @@ public function handle(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            $speakerRequest->photoPath = $this->generateUrl('index', [], UrlGeneratorInterface::ABSOLUTE_URL).'/images/default_speaker.svg';
```

Comment:
`generateUrl` is part of the `UrlGeneratorInterface`, inject that service and use it.

Reply:
`'/images/default_speaker.svg'` should not be a string but should rather come from the `Packages` service provided by Symfony Assets.